assets/terraform-modules/packet/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -16,7 +16,7 @@ module "bootkube" {
 
   # Select private Packet NIC by using the can-reach Calico autodetection option with the first
   # host in our private CIDR.
-  network_ip_autodetection_method = "can-reach=${cidrhost(var.node_private_cidr, 1)}"
+  network_ip_autodetection_method = "can-reach=${cidrhost(var.node_private_cidrs[0], 1)}"
```

Comment:
We should document, that first CIDR defined is special. Or actually have an extra parameter for extra CIDRs then...

Reply:
This is for calico to identify which CIDR to connect with other calico agents right? 

If this is the correct understanding, I think it does not matter which private CIDR is given. On EM all the node private CIDRs are within range `10.0.0.0/8` and there is only one interface with that range IP.

Do you think this can cause problem, if yes what kind?

Reply:
So you're saying it doesn't matter which CIDR smaller than /8 we put here, it should always end with the same result, as Calico will select the same interface to communicate, right? I guess that make sense. Can we add a comment explaining it then?

Reply:
I have changed it to use the first controller's private IP.